### PR TITLE
Update documentation for 1-bit Adam

### PR DESCRIPTION
**Pull request by awan-10**
This covers the following.

1. tutorial
2. news item
3. blog post
4. features page update

### CODE REVIEW

File: docs/_pages/config-json.md [hunk 1]
```diff
@@ -53,6 +53,24 @@ title: "DeepSpeed Configuration JSON"
...
+    "params": {
+      "lr": 0.001,
+      "betas": [
+        0.8,
+        0.999
+      ],
+      "eps": 1e-8,
+      "weight_decay": 3e-7
+      "freeze_step": 400
+      "cuda_aware": true
```
Comment (jeffra) [inline docs/_pages/config-json.md:70]: Missing commas

File: docs/_pages/features.md [hunk 2]
```diff
@@ -158,6 +158,16 @@ Please see the [core API doc](https://deepspeed.readthedocs.io/) for more detail
...
 
+### 1-bit Adam optimizer with up to 5x less communication
+
+DeepSpeed has an efficient implementation of a novel algorithm called 1-bit Adam.
+It offers the same convergence as Adam, incurs up to 5x less communication that enables
+up to 3.5x higher throughput for BERT-Large pretraining and up to 2.7x higher throughput
+for SQuAD fine-tuning on bandwidth-limited clusters. For more details on usage and performance,
+please refer to the detailed [tutorial](https://www.deepspeed.ai/tutorials/onebit-adam) and
+[blog post](https://www.deepspeed.ai/news/2020/09/09/onebit-adam-blog-post.md), respectively.
+<!-- **TODO: add paper link when it is ready ** -->
```
Comment (jeffra) [inline docs/_pages/features.md:169]: Should we say a paper will be coming soon that will go into even more details?

File: docs/_posts/2020-09-09-onebit-adam-blog-post.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+The idea of error compensation can be summarized as: 1) doing compression, 2) memorizing the compression error, and then 3) adding the compression error back in during the next iteration. For SGD, doing error compression leads to:
+
+<img src="https://render.githubusercontent.com/render/math?math=x_t%3D%20x_%7Bt-1%7D%20-%20%5Cgamma%20C(g_t%20%2B%20e_%7Bt-1%7D)%2C%20%5Cquad%20e_t%20%3D%20g_t%2Be_%7Bt-1%7D-C(g_t%2Be_%7Bt-1%7D%20)">
+
+Where C(⋅) is the 1-bit compression operator. The good thing about doing this error compensation is that the history compression error (e_t and e_(t-1)) would be canceled by itself eventually, which can be seen by:
+
+<img src="https://render.githubusercontent.com/render/math?math=x_t%3Dx_%7Bt-1%7D-%5Cgamma(g_t%2Be_%7Bt-1%7D-e_t%20)">
+
+
+This strategy has been proven to work for all optimization algorithms that are linearly dependent on the gradient, such as SGD and Momentum SGD.
```
Comment (jeffra) [inline docs/_posts/2020-09-09-onebit-adam-blog-post.md:33]: I feel saying "work for **all** optimization algorithms" might be a bit too strong right? maybe drop the 'all'? As in:

"This strategy has been proven to work for optimization algorithms that are linearly dependent on the gradient, such as SGD and Momentum SGD."

**Review by jeffra — APPROVED** [review]

File: docs/_pages/config-json.md (hunk 1 above)
Comment (awan-10) [reply docs/_pages/config-json.md]: Fixed.

File: docs/_pages/features.md (hunk 2 above)
Comment (awan-10) [reply docs/_pages/features.md:169]: I am not sure about this one. Paper seems a few weeks away so I suggest not to add it now.

File: docs/_posts/2020-09-09-onebit-adam-blog-post.md (hunk 3 above)
Comment (awan-10) [reply docs/_posts/2020-09-09-onebit-adam-blog-post.md]: Agreed. Fixed.

**Pull request merged**